src/Microsoft.HttpRepl/Resources/Strings.resx

```diff
@@ -264,4 +264,10 @@ When +history option is specified, commands specified in the text file will be a
     <value>If {0} is not an absolute URI, {1} must be specified.</value>
     <comment>{0} is the name of the commandSpecifiedPath parameter, {1} is the name of the baseAddress uri parameter</comment>
   </data>
-</root>
+  <data name="ListCommand_Error_NoBaseAddress" xml:space="preserve">
+    <value>No base address has been set, so there is nothing to list.</value>
```

Comment:
Is it clear (actionable) from these how to set the base address or directory structure?  `set base` seems like a fairly intuitive jump, but maybe we should be explicit just to be sure?

For directory structure, the action to resolve this would be to set swagger?  That might not be as intuitive from this wording.

Reply:
Fair. I updated both. The one for set swagger might be changing, based on my really long email and whatever comes of that discussion. But for now, this points towards set swagger.